features/raadpleeg-verblijfplaats-met-periode/in-onderzoek.feature

```diff
@@ -484,4 +484,139 @@ Functionaliteit: leveren van inOnderzoek bij raadplegen met periode
+      | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) |
+      | 0800                 | Testpad            | 1                  |
+      En adres 'A2' heeft de volgende gegevens
+      | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) |
+      | 0800                 | Testpad            | 2                  |
+      En de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20231014                           |
+      En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+      | 0800                              | 20240516                           | <aanduiding onderzoek>          | 20240526                       |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2023-07-01          |
+      | datumTot            | 2024-01-01          |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | type  | datumVan.type | datumVan.datum | datumVan.langFormaat | datumTot.type | datumTot.datum | datumTot.langFormaat | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving |
+      | Adres | Datum         | 2023-10-14     | 14 oktober 2023      | Datum         | 2024-05-16     | 16 mei 2024          | 0800                         | Hoogeloon, Hapert en Casteren        |
+      En heeft de verblijfplaats de volgende 'inOnderzoek' gegevens
+      | naam                             | waarde      |
+      | datumTot                         | true        |
+      | datumIngangOnderzoek.type        | Datum       |
+      | datumIngangOnderzoek.datum       | 2024-05-16  |
+      | datumIngangOnderzoek.langFormaat | 16 mei 2024 |
+      En heeft de verblijfplaats de volgende 'verblijfadres' gegevens
+      | naam                                         | waarde      |
+      | korteStraatnaam                              | Testpad     |
+      | huisnummer                                   | 8           |
+      En heeft de verblijfplaats de volgende 'adressering' gegevens
+      | naam                                         | waarde                        |
+      | adresregel1                                  | Testpad 8                     |
+      | adresregel2                                  | HOOGELOON, HAPERT EN CASTEREN |
+
+      Voorbeelden:
+      | omschrijving                                  | aanduiding onderzoek |
+      | hele categorie verblijfplaats                 | 80000                |
```

Comment:
in alle andere scenarios hebben we niet gespecificeerd dat aanduiding in onderzoek uit 5 cijfers kan bestaan. Is dat een requirement?